src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,61 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String CHECK_SUPPLY = "supply";
+
     public void getStatistic(String fromFileName, String toFileName) {
+        String[] dataFromFile = readFile(fromFileName);
+        String[] calculatedStatistic = calculateStatistic(dataFromFile);
+        writeFile(calculatedStatistic, toFileName);
+    }
+
+    private String[] readFile(String fromFileName) {
+        StringBuilder stringBuilder = new StringBuilder();
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(fromFileName))) {
+            String value = bufferedReader.readLine();
+            while (value != null) {
+                stringBuilder.append(value).append(" ");
```

Comment:
Space is a magic value here. Introduce a constant (DELIMITER or SEPARATOR) please instead of using hardcoded value. Think about more unique value than space, in general case we can have the sentence in file with words divided by spaces and space doesn't mean that is new line.